pkg/cmd/roachtest/tests/tpcc.go

```diff
@@ -740,6 +740,25 @@ func registerTPCC(r registry.Registry) {
 		},
 	})
 
+	r.Add(registry.TestSpec{
+		Name:              "tpcc-nowait/nodes=5/w=1000",
+		Owner:             registry.OwnerTestEng,
+		Benchmark:         true,
+		Cluster:           r.MakeClusterSpec(6, spec.CPU(16), spec.WorkloadNode()),
+		CompatibleClouds:  registry.AllClouds,
+		Suites:            registry.Suites(registry.Nightly),
```

Comment:
Also add this:

https://github.com/cockroachdb/cockroach/blob/0077599440ca290c1bb2178910e66b5c4ed24498/pkg/cmd/roachtest/tests/sysbench.go#L334
so that it's not subject to manual test selection.

Reply:
Added.  Of note is that the `TestSelectionOptOutSuites` variable is peculiarly named.  I'd think that a more intuitive name would be `AlwaysRunSuites`, since that's the end result.  The way it's named right now, you need to understand how test selection works, and what test selection "opting out" implies.  

I don't want to change this unilaterally, but pinging @nameisbhaskar, who added this.

Reply:
Seems like a good separate PR. I agree that what you're suggesting is much clearer.